frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/row_detail.dart

```diff
@@ -210,10 +211,16 @@ class _CreatePropertyButtonState extends State<_CreatePropertyButton> {
         height: 40,
         decoration: _makeBoxDecoration(context),
         child: FlowyButton(
-          text: FlowyText.medium(LocaleKeys.grid_field_newColumn.tr()),
+          text: FlowyText.medium(
+            LocaleKeys.grid_field_newProperty.tr(),
+            color: AFThemeExtension.of(context).textColor,
+          ),
           hoverColor: AFThemeExtension.of(context).lightGreyHover,
           onTap: () {},
-          leftIcon: svgWidget("home/add"),
+          leftIcon: svgWidget(
+            "home/add",
```

Comment:
Just a note, we could make this better by creating a class called `SVGRegistry` that works similarly to how `LocaleKeys` works.

```dart
class SVGRegistry {
  static String get home_add  => "home_add";
}
```

Reply:
Yes, that's a good suggestion. I kept it since it keeps a consistent way to apply the icon in current repo. We could consider to change it in the future.